zerver/views/auth.py

```diff
@@ -330,6 +330,8 @@ def login_page(request, **kwargs):
     if is_subdomain_root_or_alias(request) and settings.REALMS_HAVE_SUBDOMAINS:
         redirect_url = reverse('zerver.views.registration.find_my_team')
         return HttpResponseRedirect(redirect_url)
+    if request.user.is_authenticated():
```

Comment:
One confusion:
`if is_subdomain_root_or_alias(request) and settings.REALMS_HAVE_SUBDOMAINS:
        redirect_url = reverse('zerver.views.registration.find_my_team')
        return HttpResponseRedirect(redirect_url)
    if request.user.is_authenticated():
        return HttpResponseRedirect("/")`
Instead of the above one, you want this:
 if request.user.is_authenticated():
        return HttpResponseRedirect("/")`
`if is_subdomain_root_or_alias(request) and settings.REALMS_HAVE_SUBDOMAINS:
        redirect_url = reverse('zerver.views.registration.find_my_team')
        return HttpResponseRedirect(redirect_url)
   
 right?